module/Photo/src/Photo/View/Helper/GlideUrl.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Photo\View\Helper;
+
+use Zend\View\Helper\AbstractHelper;
+use Zend\View\Exception;
+use League\Glide\Urls\UrlBuilderFactory;
+
+/**
+ * Url view helper for generating (signed) glide url's
+ * Usage: $this->scriptUrl()->requireUrl('/url/route');
+ *
+ * @package Application\View\Helper
+ */
+class GlideUrl extends AbstractHelper
+{
+
+    protected $config;
+
+    /**
+     * @return \Photo\View\Helper\GlideUrl
+     */
+    public function __invoke()
+    {
+        if ($this->config === null)
+            throw new \Exception('No config provided to GlideUrl helper');
+        return $this;
+    }
+
+    /**
+     * Gets a signed glide URL
+     * @param $imagePath
+     * @param $params
+     * @return string
+     */
+    public function getUrl($imagePath, $params)
+    {
+        $urlBuilder = UrlBuilderFactory::create($this->config['glide']['base_url'], $this->config['glide']['signing_key']);
```

Comment:
Why recreate the builder every time? Why even create it in this class? Why isn't it injected?

Reply:
Whoops, this should be injected indeed